packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -320,7 +342,7 @@ contract ChugSplashManager is Owned {
                 // standard OOG, then this would halt the entire contract. TODO: Make sure this
                 // cannot happen in any case other than OOG.
                 require(
-                    address(created) != proxy,
+                    address(created) == proxy,
```

Comment:
Changed this check from `!=` to `==` since it seems like the addresses are supposed to match.